packages/neuron-ui/src/components/NervosDAODetail/CellsCard/index.tsx

```diff
@@ -0,0 +1,130 @@
+
+  return (
+    <div className={styles.cells}>
+      <LockInfoDialog lockInfo={showingLockInfo} isMainnet={isMainnet} onDismiss={() => setShowingLockInfo(null)} />
+      <div className={styles.cellTypes} role="tablist">
+        {tabs.map(tab => (
+          <div
+            key={tab.id}
+            className={clsx(styles.cellType, { [styles.active]: selectedTab.id === tab.id })}
+            role="tab"
+            tabIndex={0}
+            onKeyDown={onEnter(() => onTabChange(tab.id))}
+            onClick={() => onTabChange(tab.id)}
+          >
+            {tab.title}
+          </div>
+        ))}
+      </div>
+      <div className={styles.cellsPanel} role="tabpanel">
+        <div className={clsx(styles.row, styles.columns)}>
+          <div className={styles.column}>{t('nervos-dao-detail.index')}</div>
+          <div className={styles.column}>{t('nervos-dao-detail.address')}</div>
+          <div className={styles.column}>
+            {t('nervos-dao-detail.amount')}
+            {isPrivacyMode ? (
+              <EyesClose className={styles.eyes} onClick={() => setIsPrivacyMode(false)} />
+            ) : (
+              <EyesOpen className={styles.eyes} onClick={() => setIsPrivacyMode(true)} />
+            )}
+          </div>
+        </div>
+        {selectedTab.cells.map((cell, index) => {
+          const address = cell.lock != null ? scriptToAddress(cell.lock, isMainnet) : null
+          const capacity = cell.capacity ? shannonToCKBFormatter(cell.capacity) : '--'
+
+          return (
+            // eslint-disable-next-line react/no-array-index-key
+            <div key={index} className={styles.row}>
+              <div>{index}</div>
+              <div>
+                {!address ? (
+                  t('transaction.cell-from-cellbase')
+                ) : (
+                  <>
+                    <span className={styles.address}>{`${address.slice(0, 20)}...${address.slice(-20)}`} </span>
+                    <ScriptTag isMainnet={isMainnet} script={cell.lock} onClick={() => setShowingLockInfo(cell.lock)} />
+                  </>
+                )}
+              </div>
+              <div>{`${isPrivacyMode ? '******' : capacity} CKB`}</div>
+            </div>
+          )
+        })}
+      </div>
+    </div>
+  )
+}
+
+export const CellsCard: FC<{ transaction: State.DetailedTransaction }> = ({ transaction }) => {
+  const [t] = useTranslation()
+
+  const inputsTitle = useMemo(
+    () => `${t('transaction.inputs')} (${transaction.inputs.length}/${localNumberFormatter(transaction.inputsCount)})`,
+    [transaction.inputs.length, transaction.inputsCount, t]
+  )
+
+  const outputsTitle = useMemo(() => {
+    return `${t('transaction.outputs')} (${transaction.outputs.length}/${localNumberFormatter(
+      transaction.outputsCount
+    )})`
+  }, [transaction.outputs.length, transaction.outputsCount, t])
+
+  const tabs = useMemo(
+    () => [
+      {
+        id: 'inputs',
+        title: inputsTitle,
```

Comment:
I think we can directly get the `inputsTitle` here. What do you think?

Reply:
resolved